src/app/i18n/locales/zh.json

```diff
@@ -1540,7 +1540,18 @@
         }
       },
       "teams": {
-        "title": "团队"
+        "title": "團隊",

```

Comment:
these ch translations are exactly the same as those of tw. Is this correct?

Reply:
I've changed the tw translation